lib/cisco_node_utils/feature.rb

```diff
@@ -169,6 +169,16 @@ def self.private_vlan_enabled?
     end
 
     # ---------------------------
+    def self.tacacs_enable
+      return if tacacs_enabled?
```

Comment:
What do you think of this?

```
-    return if tacacs_enabled?
+    return if tacacs_enabled? || platform == :ios_xr
```

...what you have is okay, I'm just wondering if the platform checks would be better in the feature enable methods themselves instead of sprinkling plat checks in each of the callers. 

Maybe leave a TBD comment if you don't want to address it now.